src/log-tlslog.c

```diff
@@ -451,12 +451,12 @@ static int LogTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
         return 0;
     }
 
+    MemBufferReset(aft->buffer);
```

Comment:
May I suggest that (1) The call to `MemBufferReset` be retained in `LogTlsLogger` and (2) a call to `MemBufferReset` be added in `LogTlsLogCustom`?

Reply:
After reading the code once more, I understand the reasoning why you want this change. I don't like how the code in _log_tlslog.c_ is structured. I would prefer if the code was structured as in _output-json-tls.c_ where functions for logging 'basic', 'extended' and 'custom' just fills in the buffer, and all the other functionality is shared. Would you be okay with that I make this change for tls-log also instead? :)

Reply:
You have a valid point and yes, I'm ok with the original proposed change.

Reply:
I've made the changes I mentioned in #4232 :)